notebooks/env.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<p>You need to give an example of how to use this (eg configs). Also what to do when they have a custom environment that is a gym environment (eg registering with gym and then using hive).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chandar-lab/RLHive/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I'm not sure if I understand your point here. What exactly do you mean by "example of how to use this"? We'll use the environment we've created in the following subsections. If you're only talking about&nbsp;custom environments and registering them, don't you think moving them up would make the tutorial a bit confusing?</p>